docs/Configuring_the_native_Android_SIP_client.md

```diff
@@ -24,11 +26,13 @@ list.
     -   Set **Authentication username** to
         "sip:*&lt;username\>*@*&lt;server\>*", substituting (note: once we
         switch to storing user digest by private id, this will change)
-
+    -   Some SIP clients do not have an **Authentication username** field. To
+        make calls through Clearwater using these clients you will have to
+        configure an external HSS to [allow one subscriber to have two private
```

Comment:
It's useful to say *why* - "one private identity of the form 1234@example.com (the IMS standard form), and one private identity of 1234 (which the phone will default to)".